package.json

```diff
@@ -129,21 +129,21 @@
         "bootstrap": "3.3.7",
         "bootstrap-slider": "10.6.2",
         "bowser": "1.0.0",
-        "corejs-typeahead": "1.0.1",
+        "corejs-typeahead": "^1.3.1",
         "d3": "3.5.6",
-        "datatables.net": "1.11.3",
+        "datatables.net": "^1.11.5",
         "datatables.net-bs": "^1.11.5",
         "datatables.net-buttons-bs": "1.4.2",
         "dropzone": "5.7.0",
         "file-saver": "1.3.4",
         "font-awesome": "4.7.0",
         "google-code-prettify": "1.0.5",
-        "handlebars": "4.0.5",
+        "handlebars": "^4.7.7",
         "jquery": "2.2.4",
         "jquery-nearest": "1.3.1",
-        "jquery-ui": "1.12.1",
+        "jquery-ui": "^1.13.1",
         "jquery.tipsy": "1.0.3",
-        "js-yaml": "3.3.1",
+        "js-yaml": "3.14.1",
```

Comment:
We use this module for the `requirejs-yaml` plugin (now archived/unmaintained); `js-yaml` is now on v4.x, but `requirejs-yaml` uses the 3.x API, so we cannot update any further

Reply:
👍